index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul hasan portfolio</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <style> body{background: url("image/pic.png");} </style>
+
+  </head>
+
+  <body>
+    <header>
+    <nav>
+    <div class="navbar">
+            <h1 class="logo"> My Logo </h1>
```

Comment:
- [ ] Good implementing the logo. I suggest you set it above the `nav` tag and replace the `h1` with an `a` tag. Normally, a site's logo will enable a user to navigate back to the homepage. It will be good if you could fix this.

See example
```
<header>
   <a> // you logo </a>
   <nav> // your hamburger icon </nav>
</header/>
```